docs.json

```diff
@@ -60,6 +60,14 @@
                 "disable_filters": true,
                 "disable_summary_gen": true
             }
+        },
+        {
+          "name": "doxygen_to_xml",
+          "options": {
+            "INPUT": "libraries/chain/include/eosio/chain/webassembly/interface.hpp"
+          },
+          "disable_default_filters" :true,
+          "filters" : []
```

Comment:
Need to add the "doxybook" generator in "docs.json" (after the "doxygen_to_xml" generator) for the contents to render on the left navigation menu:
```
            ...
        },
        {
            "name": "doxybook",
            "options": {}
        }
```

Reply:
done